comms/storage/tests/upload_test.go

```diff
@@ -0,0 +1,88 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	"comms.audius.co/shared/utils"
+	"golang.org/x/exp/slices"
+)
+
+func TestAudioUpload(t *testing.T) {
+	client, server, err := SpawnServer()
```

Comment:
This doesn't work right now because the system isn't designed for the server to be spun up and down. I wanna start a discussion on this so this _can_ work